docs/api.rst

```diff
@@ -25,6 +25,5 @@
 API Docs
 ========
 
-invenio_deposit
----------------
-
+.. automodule:: invenio_deposit.api
```

Comment:
@jirikuncar 
Sphinx said me: 
```
WARNING: py:data reference target not found: invenio_records_rest.config.RECORDS_REST_DEFAULT_SORT
```

But, actually, this configuration is not documented online for now (see my https://github.com/inveniosoftware/invenio-records-rest/pull/105 ).
Is it why sphinx returns the warning?